.github/workflows/fastapi.yml

```
@@ -0,0 +1,40 @@
+name: fastapi-testsuite
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  fastapi-test:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: fastapi
+    strategy:
+      matrix:
+        python-version: [3.9]
+      fail-fast: false
+    steps:
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
```

```suggestion
    steps:
      - uses: actions/setup-python@v2
        with:
          python-version: '3.9'
```
This way, we can also get rid of the `strategy: ...` block as well and make this simpler 😄

---

Updated 😄